### ISSUE TITLE
Firework fires while flying with elytra in freecam

### ISSUE THREAD

**Issue by HeavinSlayer109** [hashalite/Freecam#72]
When I was flying in my elytra while holding a firework, I went to freecam, I pressed right click for some unknown reason, but it fired the firework. Thats it

**Comment by MattSturgeon** [comment]
I believe this is intended behaviour.

"Allow Interaction" and "Interaction Mode" relate to interacting with blocks/entities in the world, but there's no setting that relates to consumption of items.

We could consider this a feature request for adding a config option that completely disables using held items, I guess 🤷

**Comment by hashalite** [comment]
Yes, this is intended behavior. I may make a separate config option for item consumption in the future.
